src/plugins/intel_npu/src/plugin/npuw/compiled_model.hpp

```diff
@@ -124,6 +124,8 @@ class CompiledModel : public ov::ICompiledModel {
         ov::SoPtr<ov::ICompiledModel> ref_compiled_model;
         bool switched_to_ref = false;
 
+        std::optional<bool> unpack_required = std::nullopt;
```

Comment:
`unpack_required` is an individual flag per every tensor in the closure, so in our current approach I'd expect it to be `std::vector<bool>` - same as `closure`, `scales`, and `zerops`.

Reply:
Also it seems it is not used at this point

Reply:
Reworked and done